ledger/src/snapshot_utils.rs

```diff
@@ -30,7 +31,13 @@ pub const TAR_SNAPSHOTS_DIR: &str = "snapshots";
 pub const TAR_ACCOUNTS_DIR: &str = "accounts";
 pub const TAR_VERSION_FILE: &str = "version";
 
-pub const SNAPSHOT_VERSION: &str = "1.0.0";
+pub const SNAPSHOT_VERSION_1_0: &str = "1.0.0";
+pub const SNAPSHOT_VERSION_1_1: &str = "1.1.0";
```

Comment:
This can just be `SNAPSHOT_VERSION` ("Current" is the default, only name the legacy versions)